hack/fuse-demo/coord-app-pg.Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM debian
```

Comment:
We should use a release so that the image does not change. This is important for the PG version management.